src/model/trust.model.ts

```diff
@@ -79,6 +79,10 @@ export interface TrustIndividual {
   sa_address_postal_code?: string;
   sa_address_care_of?: string;
   sa_address_po_box?: string;
+  is_individual_still_involved_in_trust?: string,
```

Comment:
@mattch1 has named this field as just `still_involved` in his PR. I think it would be good to be consistent and given this field (like all the others here) are in the `TrustIndividual` interface I think just `still_involved` is fine.

Reply:
ok, i'll change it back.

Reply:
changed back to use still_involved